cmd/utils.go

```diff
@@ -271,3 +271,25 @@ func jsonLoadFromSeeker(r io.ReadSeeker, data interface{}) error {
 	}
 	return json.NewDecoder(r).Decode(data)
 }
+
+// Save to fs format.json
+func jsonSaveToSeeker(f interface {
```

Comment:
Since the seek offset is not being passed in, it would be better to name this function something like `replaceFileWithJSON` as it more closely reflects what the function does.

Reply:
renamed to simpler function names:
jsonLoadFromSeeker -> jsonLoad
jsonSaveToSeeker -> jsonSave
( Since we also have similar function names quick.Load() and quick.Save() )